### DISCUSSION TITLE
Fail Animation doesn't look right with Low Background Dim

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16046] [Q&A] [answered]
Video:

https://user-images.githubusercontent.com/64581009/145693208-4bf58c54-bbcb-4003-90c3-6996b02b400b.mp4

Unsure if this is intended or not but: 
_It should animate the entire background with it._ Could look nicer.
or
_Remove the black box._ Works both ways.

Despite that, I'm sure it's not really important(?) since I don't think most players would use background dim lower than 80%.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #16066.
